tests/containers/bci_prepare.pm

```diff
@@ -53,20 +53,20 @@ sub packages_to_install {
         push @packages, ('python3-virtualenv') if ($bci_virtualenv);
     } elsif ($host_distri eq 'sles' || $host_distri =~ /leap/) {
         my $version = "$version.$sp";
-        push @packages, ('python3-virtualenv') if ($bci_virtualenv);
         if ($version =~ /12\./) {
             script_retry("SUSEConnect --auto-agree-with-licenses -p sle-sdk/$version/$arch", delay => 60, retry => 3, timeout => $scc_timeout);
             # PackageHub is needed for jq
             script_retry("SUSEConnect -p PackageHub/12.5/$arch", delay => 60, retry => 3, timeout => $scc_timeout);
             script_retry('zypper -n in jq', retry => 3);
-            # Note tox is not available on SLES12
-            push @packages, qw(python36-pip);
-            die "virtualenv is not supported on 12-SP5" if ($bci_virtualenv);
+            assert_script_run("zypper ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/Update:/Products:/SaltBundle:/Update/standard/ saltbundle");
+            assert_script_run("zypper -n install venv-salt-minion");
         } elsif ($version =~ /15\.[1-3]/) {
+            push @packages, ('python3-virtualenv') if ($bci_virtualenv);
```

Comment:
Why is this needed? So far I don't see any problems without this.

Also: If we're changing this, then you also need to add VRs for those SP.

Reply:
Agree,d but then we need VRs for all affected OS systems. For BCI they are important, as causing issues on the openQA level needs a lot of restarting.

Let's be safe here.